cronjobs/scripts/price_entrypoint.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
entrypoint was unified., we do not need this file anymore.

Reply:
Removed thanks!